packages/core/core/src/worker.js

```diff
@@ -1,45 +1,43 @@
 // @flow strict-local
 
-import type {ParcelOptions, AssetRequest, JSONObject} from '@parcel/types';
+import type {ParcelOptions, AssetRequest} from '@parcel/types';
 import type {Bundle} from './types';
 import type BundleGraph from './BundleGraph';
 
 import TransformerRunner from './TransformerRunner';
 import PackagerRunner from './PackagerRunner';
 import Config from './Config';
 
-type Options = {|
-  config: Config,
-  options: ParcelOptions,
-  env: JSONObject
-|};
-
-let transformerRunner: TransformerRunner | null = null;
-let packagerRunner: PackagerRunner | null = null;
+export function init() {}
```

Comment:
What do you think about getting rid of `init` entirely? Right now `WorkerFarm` only has one usecase and that's this worker file, and it doesn't use `init`.

Reply:
I'd be ok with it.